styel.css

```diff
@@ -1,114 +1,395 @@
+  margin-left: 160px;
+}
+
+.lang-list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+  padding: 12px;
+  gap: 12px;
+  width: 327px;
+}
+
+.js {
+  background: #f7f7f9;
+  border-radius: 8px;
+  width: 303px;
   font-weight: 500;
-  font-size: 16px;
-  line-height: 24px;
-  text-decoration: none;
+  font-size: 15px;
+  line-height: 20px;
+  font-family: inherit;
+  display: flex;
+  align-items: center;
 }
 
-.img {
-  width: 170.17px;
-  height: 20px;
-  left: 260px;
-  top: 642px;
+.frame {
+  width: 327px;
+  height: 72px;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #000;
 }
 
-.img img {
-  margin-right: 10px;
-  margin-top: 5px;
+.frame img {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  width: 40px;
+  right: 40px;
+  top: calc(50% - 40px / 2);
+  border-radius: 8px;
+  margin-left: 160px;
 }
 
-img a:hover {
-  color: #6070ff;
+.skills {
+  width: 327px;
+  height: 72px;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #000;
 }
 
-.img ul {
-  list-style: none;
+.skills img {
   display: flex;
   flex-direction: row;
-  flex-wrap: wrap;
-  padding-left: 0;
+  align-items: flex-start;
+  padding: 8px;
+  width: 40px;
+  right: 40px;
+  top: calc(50% - 40px / 2);
+  border-radius: 8px;
+  margin-left: 230px;
+}
+
+.sep {
+  height: 1px;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: #dfe1e6;
+}
+
+.contact-me {
+  background: #6070ff;
+  background-image: url("shapes.png");
+  background-repeat: no-repeat;
+  background-position: 160px 40px;
+  border-top-left-radius: 60px;
+}
+
+.title-contact-me {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+}
+
+.contact-me p {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+input {
+  background: #fff;
+  border-radius: 8px;
+  border: none;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  align-items: center;
+  color: #172b4d;
+  margin-left: 30px;
+  height: 50px;
+  width: 270px;
+  margin-top: 20px;
+  font-family: "Poppins", sans-serif;
+}
+
+textarea {
+  background: #fff;
+  border-radius: 8px;
+  border: none;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  align-items: center;
+  color: #172b4d;
+  margin: 30px;
+  margin-top: 20px;
+  font-family: inherit;
+}
+
+.sub {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: flex-start;
+  letter-spacing: 0.03em;
+  border: none;
+  color: #6070ff;
+  font-family: inherit;
+  padding: 15px;
+  margin-left: 30px;
+  border-radius: 10px;
+  padding-right: 0;
+  margin-right: 150px;
 }
 
-.img ul li {
-  display: inline-block;
+.sub:hover {
+  background-color: #6070ff;
+  color: #fff;
 }
```

Comment:
- [ ] Kindly make sure to add interactions like enabled, hover, pressed for all the buttons.
<img width="389" alt="Screenshot 2022-08-20 at 8 03 56 PM" src="https://user-images.githubusercontent.com/70269507/185758836-617a0a13-85d9-47dd-90e3-2cec54f42898.png">